open_spiel/games/phantom_go/phantom_go_board.h

```diff
@@ -111,18 +113,20 @@ class Neighbours4 {
 // It only implements the minimum of functionality necessary to support the
 // search and is optimized for speed and size. Importantly, it fits on the
 // stack. For detailed numbers, run the benchmarks in go_board_test.
-class GoBoard {
+class PhantomGoBoard {
  public:
-  explicit GoBoard(int board_size);
+  explicit PhantomGoBoard(int board_size);
 
   void Clear();
 
-  std::vector<GoColor> observationWhite;
-  std::vector<GoColor> observationBlack;
+  
 
-  std::pair<int, int> stoneCount;
+  std::array<int, 2> getStoneCount() const { return stoneCount; };
```

Comment:
Rewrite as using the color enum, and then you don't need to use the (uint8_t) casting everywhere.

Reply:
And add checks for the color being either black or white